### PR TITLE
Add Nois Network Mainnet - Brings fair randomness to the interchain

### PR DESCRIPTION
**Pull request by kaisbaccour**

### CODE REVIEW

File: cosmos/nois.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "rpc": "https://rpc-nois.sr20de.xyz/",
+  "rest": "https://api.noismain.mcbnode.online",
```
Comment (webmaster128) [inline cosmos/nois.json:3]: Should we add the proxy endpoints here?

```suggestion
  "rpc": "https://rpc.cosmos.directory/nois",
  "rest": "https://rest.cosmos.directory/nois",
```

**Comment by HeesungB** [comment]
@kaisbaccour 
Failed to get response /status from rpc endpoint.
Could you check your rpc endpoint?

**Comment by kaisbaccour** [comment]
> @kaisbaccour Failed to get response /status from rpc endpoint. Could you check your rpc endpoint?
@HeesungB 
I just switched to polkachu (another rpc endpoint). Should be better now.

**Comment by kaisbaccour** [comment]
Hey @HeesungB Any idea how we could solve this ? Thanks a lot

**Comment by HeesungB** [comment]
@kaisbaccour 
If you use this for `https://nois-rpc.nysa.network`, it works.
Do you guys have a node up and running?
You must enter `node provider information` and change the image size to 256x256.

**Comment by kaisbaccour** [comment]
@HeesungB We don't run as Nois devs nodes that are up all times.However most validators do.
I made a commit to 
- change the rpc to `https://nois-rpc.nysa.network`.
- added the `node provider info`
- changed the image size to 256x256

**Comment by HeesungB** [comment]
@kaisbaccour @webmaster128 Your PR merged.
Thank you for contribution!

**Pull request merged**